app/static/js/createEvents.js

```diff
@@ -60,18 +60,86 @@ function format24to12HourTime(timeStr){
       });
 
   }
+
+
+function getSelectedFiles(){
+  let _fileHolder = new DataTransfer();
+  $(".fileHolder").each(function(){
+    _fileHolder.items.add($(this).data("file"));
+  });
+  return _fileHolder.files;
+}
+
+
+function hasUniqueFileName(fileName){
+  let nameHolders = $(".fileName").toArray();
+  for (let i = 0; i < nameHolders.length; i++){
+    if (fileName == $(nameHolders[i]).data("filename")) return false;
+  }
+  return true;
+}
```

Comment:
it might be simpler for this function to use something like the `.filter` method on `$('.fileName')` to get the # of filenames that match and then return whether that result is 0 length or not

Reply:
Even better, you don't need the `if` and separate return statements. Just return that conditional directly.